From c15887bbc9344317389dd4134aa5f5a1d6d13fca Mon Sep 17 00:00:00 2001
Message-Id: <patch-2.6-c15887bbc93-20210510T104937Z-avarab@gmail.com>
In-Reply-To: <cover-0.6-00000000000-20210510T104937Z-avarab@gmail.com>
References: <cover-0.4-00000000000-20210505T121857Z-avarab@gmail.com>
	<cover-0.6-00000000000-20210510T104937Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Tue, 6 Apr 2021 16:24:13 +0200
Subject: [PATCH v2 2/6] Makefile: don't re-define PERL_DEFINES
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Since 07d90eadb50 (Makefile: add Perl runtime prefix support,
2018-04-10) we have been declaring PERL_DEFINES right after assigning
to it, with the effect that the first PERL_DEFINES was ignored.

This didn't matter in practice since the first line had all the same
variables as the second, so we'd correctly re-generate everything. It
just made for confusing reading. Let's remove that first assignment.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile b/Makefile
index ad618cea33f..ea387b431e1 100644
--- a/Makefile
+++ b/Makefile
@@ -2270,8 +2270,6 @@ perl_localedir_SQ = $(localedir_SQ)
 
 ifndef NO_PERL
 PERL_HEADER_TEMPLATE = perl/header_templates/fixed_prefix.template.pl
-PERL_DEFINES = $(PERL_PATH_SQ):$(PERLLIB_EXTRA_SQ):$(perllibdir_SQ)
-
 PERL_DEFINES := $(PERL_PATH_SQ) $(PERLLIB_EXTRA_SQ) $(perllibdir_SQ)
 PERL_DEFINES += $(RUNTIME_PREFIX)
 
-- 
2.31.1.838.g924d365b763

